mlserver/types/__init__.py

```diff
@@ -45,4 +47,8 @@
     "RepositoryIndexResponse",
     "RepositoryLoadErrorResponse",
     "RepositoryUnloadErrorResponse",
+    # Tracing
+    "Tracepoint",
+    "ArgStatus",
+    "MAX_TRACEPOINT_ARGS",
```

Comment:
I'd move these types into the `mlserver.sys_tracing` module to keep them closer to their own scope.

Reply:
Those types were in `mlserver.sys_tracing` initially. The problem is that it creates a circular dependency; I'll see if I can sort it out more elegantly.

Depending on what we decide to do on the Settings side (and whether to simplify or not the available settings) this could be solved trivially.